src/sortStudents.ts

```diff
@@ -2,6 +2,84 @@
 // create and export SortType enum
 // create SortOrder type
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+interface Student {
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+}
+
+export enum SortType {
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+type SortOrder = 'asc' | 'desc';
+
+function average(grades: number[]): number {
+  return grades.reduce((a: number, b: number) => a + b) / grades.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyStudents: Student[] = [...students];
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      if (order === 'asc') {
```

Comment:
```suggestion
      if (order === 'asc') {
```
Use enum instead of string